sdk/documentintelligence/azure-ai-documentintelligence/samples/sample_analyze_custom_documents.py

```diff
@@ -60,7 +60,7 @@ def print_table(header_names, table_data):
 
 def analyze_custom_documents(custom_model_id):
     # For the Form_1.jpg, it should be the test file under the traning dataset which storage at the Azure Blob Storage path
-    # combined by DOCUMENTINTELLIGENCE_STORAGE_CONTAINER_SAS_URL and DOCUMENTINTELLIGENCE_STORAGE_PREFIX, 
+    # combined by DOCUMENTINTELLIGENCE_STORAGE_ADVANCE_CONTAINER_SAS_URL and DOCUMENTINTELLIGENCE_STORAGE_PREFIX, 
```

Comment:
why are we updating the endpoint name in the samples?

Reply:
We are using different sample data which include both dynamic and fixed tables.